_schema.graphql

```diff
@@ -36,13 +36,34 @@ type Collection {
 
   # Collection has prioritized connection to artist
   is_featured_artist_content: Boolean!
+
+  # Array of top merchandisabile artworks and associated image
+  featuredArtworks: [CollectionFeaturedArtwork!]
 }
 
 type CollectionCategory {
   name: String!
   collections: [Collection!]!
 }
 
+type CollectionFeaturedArtwork {
+  _id: ID
+  id: String
+
+  # Artwork image data
+  image: CollectionFeaturedArtworkImage!
+}
+
+type CollectionFeaturedArtworkImage {
+  aspectRatio: Int
+  height: Int
+  width: Int
+  position: Int
+  url: String
+  imageUrl: String
```

Comment:
Added these as camelCase, but we may want to reconsider because we are manually renaming the metaphysics return values for `aspect_ratio` and `image_url`.

Reply:
It makes sense to do so as that's what we're currently doing for new fields in metaphysics